molecule/docker-oracle-11/molecule.yml

```diff
@@ -29,5 +28,7 @@ verifier:
   options:
     verbose: true
   directory: ../resources/tests/
+  additional_files_or_dirs:
```

Comment:
Why additional files?

Reply:
I did it specifically, i thought it would be better to store separate tests in different files.